bin/README.md

```diff
@@ -0,0 +1,22 @@
+Sync tool for Behat Transliterator
```

Comment:
This info would be better suited in a ``CONTRIBUTING.md`` file, in a section named "Updating data" or something like that